data/language/nl-NL.yml

```diff
@@ -2191,3 +2191,209 @@ strings:
   2136: "{COLOUR WINDOW_2}Zoomen op aanwijzerpositie"
   2137: "{SMALLFONT}{COLOUR BLACK}Als dit ingeschakeld, zal er worden ingezoomd op de locatie van de aanwijzer, in plaats van in het midden van het scherm."
   2138: "{COLOUR WINDOW_2}Muziek afspelen op titelscherm"
+  2139: "Terug naar menu"
+  2140: "OpenLoco afsluiten"
+  2141: "{COLOUR WINDOW_2}Schakel AI-bedrijven uit"
+  2142: "{SMALLFONT}{COLOUR BLACK}Dit schakelt het 'nadenken' van AI-bedrijven uit, wat ze direct stillegt.{NEWLINE}In nieuwe spellen voorkomt het ook dat AI-bedrijven worden opgezet."
+  2143: "{COLOUR WINDOW_2}Hoeveelheid autosaves:"
+  2144: "{COLOUR WINDOW_2}Frequentie van autosaves:"
+  2145: "Nooit"
+  2146: "Elke maand"
+  2147: "Elke {INT32} maanden"
+  2148: "{COLOUR WINDOW_2}Bron van hoogtemap:"
```

Comment:
Ik zou kiezen voor "heightmap" of "hoogtekaart" (mijn voorkeur ligt bij die laatste). "hoogtemap" is niet heel erg duidelijk, denk ik.

Reply:
Aha! Ik was voor hoogtemap gegaan omdat de Nederlandse Wikipedia dat ook als paginatitel gebruikt. Ben het je eens dat 'map' wat ambigu is.

Reply:
Ik heb het aangepast naar "hoogtekaart".